runtime/domain.c

```diff
@@ -390,6 +420,16 @@ int caml_reallocate_minor_heap(asize_t wsize)
     (/* initialized minor heap */
       domain_state->young_start == (value*)domain_self->minor_heap_area
       && domain_state->young_end <= (value*)domain_self->minor_heap_area_end));
+}
+
+void caml_free_minor_heap() {
```

Comment:
```suggestion
Caml_inline void free_minor_heap() {
```

Reply:
Is there a reason to `inline` this? It does not sound performance-critical. Would `extern` be enough?

Reply:
You are right. Perhaps `inline` is an overkill. `static void` since the function is not used outside this file? Same for my other suggestions.

Reply:
@sabine changed this to `static`, which I think is better here.